test/__init__.py

```diff
@@ -718,3 +717,10 @@ def test_cases(suite):
             # unittest.TestSuite
             for case in test_cases(suite_or_case):
                 yield case
+
+
+def clear_warning_registry():
```

Comment:
Can you add a comment with a link to the python bug tracker?

Reply:
Done.